### PR TITLE
Change clusterrolebinding namespace by install ns

### PR DESCRIPTION
**Pull request by qiujian16**

### CODE REVIEW

File: pkg/addon/manifests/chart/templates/cluster_role_binding.yaml [hunk 1]
```diff
@@ -2,7 +2,11 @@
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
+  {{- if eq .Release.Namespace "open-cluster-management-agent-addon" }}
   name: {{ .Values.org }}:{{ template "work-manager.fullname" . }}
+  {{- else }}
+  name: {{ .Values.org }}:{{ template "work-manager.fullname" . }}:{{ .Release.Namespace }}
```
Comment (zhujian7) [inline pkg/addon/manifests/chart/templates/cluster_role_binding.yaml:8]: Is it possible to exceed the length limit?

Comment (qiujian16) [reply pkg/addon/manifests/chart/templates/cluster_role_binding.yaml:8]: the name is 253, while the namespace limit is 64

**Pull request merged**